cfc-testing/scripts/compliance-report-generator.py

```
@@ -0,0 +1,1260 @@
+
+    def _parse_junit_xml(self):
+        """Parse JUnit XML results for detailed test information."""
+        xml_file = self.cloudforge_api_dir / "target" / "surefire-reports" / \
+                   "TEST-com.cloudforgeci.api.integration.deployment.TruthTableValidationTest.xml"
+
+        if not xml_file.exists():
+            print(f"   ⚠️  JUnit XML not found: {xml_file}")
+            return
+
+        try:
+            tree = ET.parse(xml_file)
```

Ruff (0.14.8)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)